Yamb/CustomUIElements/CustomLoginTextField.swift

```diff
@@ -14,6 +14,9 @@ class CustomLoginTextField: UITextField {
         self.layer.borderWidth = 1
         self.layer.borderColor = UIColor.black.cgColor
         self.placeholder = placeholder
```

Comment:
Consider if self.autocapitalizationType = .none should be applied to every instance ?
If it is only for email, pass & confirm pass, change conditions like this :

if placeholder == email {
     self.autocapitalizationType = .none
}

if placeholder == pass || placeholder == confirmPass {
     self.isSecureTextEntry = true
     self.autocapitalizationType = .none
}